language/vm/src/deserializer.rs

```diff
@@ -819,7 +819,9 @@ fn load_signature_tokens(cursor: &mut Cursor<&[u8]>) -> BinaryLoaderResult<Vec<S
 }
 
 /// Deserializes a `SignatureToken`.
-fn load_signature_token(cursor: &mut Cursor<&[u8]>) -> BinaryLoaderResult<SignatureToken> {
+pub(crate) fn load_signature_token(
```

Comment:
I really don't want to expose this function (and a bunch of other stuff), but I don't see how otherwise I'll be able to test it in a reasonable way. Please let me know if you have a better solution.